stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3730 @@
+{
```

Comment:
<p>In case you do decide to keep the first paragraph, I'd suggest changing the github link to a hidden link (e.g. "the most popular was the community <a href="https://github.com/acupoftee/Overwatch-League-API-Documentation" target="_blank"><em>Overwatch League API</em></a>..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Good idea, but deleted the whole paragraph</p><p> </p>